src/Solvers/matrix_solver_utils.jl

```diff
@@ -95,3 +95,124 @@ end
 @inline validate_laplacian_size(N, dim) = dim == true ? N : 1
   
 @inline ensure_diagonal_elements_are_present!(A) = fkeep!(A, (i, j, x) -> (i == j || !iszero(x)))
+
+"""
+    make_column(f::Field)
+
+Return the column vector with the interior of a field `f`.
+"""
+function make_column(f::Field)
```

Comment:
Should we extend `vec` for this?

Docs for `vec`:

```
help?> vec
search: vec Vector VecOrMat VecElement BitVector DenseVector DenseVecOrMat StridedVector StridedVecOrMat AbstractVector AbstractVecOrMat InvalidStateException

  vec(a::AbstractArray) -> AbstractVector

  Reshape the array a as a one-dimensional column vector. Return a if it is already an AbstractVector. The resulting array shares the same underlying data as a, so it
  will only be mutable if a is mutable, in which case modifying one will also modify the other.
```

Reply:
sounds like we should!

Reply:
```
julia> grid = LatitudeLongitudeGrid(arch, size = (3, 3, 3),
                                    longitude = (0, 60),
                                    latitude = (15, 75),
                                    z = (-1, 0), halo=(3, 3, 4))
3×3×3 LatitudeLongitudeGrid{Float64, Bounded, Bounded, Bounded} on CPU with 3×3×4 halo and with precomputed metrics
├── longitude: Bounded  λ ∈ [0.0, 60.0]  regularly spaced with Δλ=20.0
├── latitude:  Bounded  φ ∈ [15.0, 75.0] regularly spaced with Δφ=20.0
└── z:         Bounded  z ∈ [-1.0, 0.0]  regularly spaced with Δz=0.333333

julia> c = CenterField(grid)
3×3×3 Field{Center, Center, Center} on LatitudeLongitudeGrid on CPU
├── grid: 3×3×3 LatitudeLongitudeGrid{Float64, Bounded, Bounded, Bounded} on CPU with 3×3×4 halo and with precomputed metrics
├── boundary conditions: FieldBoundaryConditions
│   └── west: ZeroFlux, east: ZeroFlux, south: ZeroFlux, north: ZeroFlux, bottom: ZeroFlux, top: ZeroFlux, immersed: ZeroFlux
└── data: 9×9×11 OffsetArray(::Array{Float64, 3}, -2:6, -2:6, -3:7) with eltype Float64 with indices -2:6×-2:6×-3:7
    └── max=0.0, min=0.0, mean=0.0

julia> vec(c) == reshape(interior(c), 3*3*3)
true
```